janus_integration/mix.exs

```diff
@@ -0,0 +1,33 @@
+defmodule JanusIntegrationTests.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :janus_integration,
+      version: "0.1.0",
+      elixir: "~> 1.10",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_), do: ["lib"]
+
+  defp deps do
+    [
+      {:elixir_janus, path: "../", override: true},
+      {:elixir_janus_transport_ws, github: "software-mansion-labs/elixir-janus-transport-ws"},
+      {:elixir_janus_plugin_videoroom,
```

Comment:
Is this really needed? I don't see any call using it. You should either test it as well or remove from deps (maybe the second option would be better if done together with adding similar test to `elixir_janus_plugin_videoroom`)